site/source/docs/getting_started/downloads.rst

```diff
@@ -198,8 +202,21 @@ The package manager can do many other maintenance tasks ranging from fetching sp
 .. _downloads-uninstall-the-sdk:
 
 Uninstalling the Emscripten SDK
-========================================================
+===============================
+
+If you want to remove the whole SDK, just delete the directory containing the
+SDK.
+
+It is also possible to :ref:`remove specific tools in the SDK using emsdk
+<emsdk-remove-tool-sdk>`.
+
+Using the Docker image
+======================
+
+The entire Emscripten SDK is also available in the form a `docker image
+<https://hub.docker.com/r/emscripten/emsdk>`_.  For example::
 
-If you want to remove the whole SDK, just delete the directory containing the SDK.
+  docker run --rm -v $(pwd):/src -u $(id -u):$(id -g) \
```

Comment:
```suggestion
  docker run --rm -v $(pwd):/src \
```

just to keep the example simpler, I wouldn't bother with permissions here. It should work with defaults too.

Reply:
The side effect of running the command with `-u` is a `helloworld.js` owned by root in the current directory which you then need to use `sudo` to remove.   There are probably other ways to avoid that, but this is the current method that we document elsewhere.

Reply:
> running the command with `-u`

Did you mean "without" here? Interesting, haven't noticed any issues so far - we don't use `-u` in Squoosh and I was able to remove files without `sudo` every time.

Reply:
If you run this command what do the permissions and ownership look like on `helloworld.js`.  On my machine its owned by root unless I add `-u`.   Maybe docker version thing?  

```
$ docker -v
Docker version 19.03.5, build 633a0ea838
$ docker run --rm -v $(pwd):/src emscripten/emsdk emcc helloworld.cpp -o helloworld.js
$ ll helloworld.js 
-rw-r--r-- 1 sbc primarygroup 114K Aug  2 17:58 helloworld.js
$ ll helloworld.js 
-rw-r--r-- 1 root root 114K Aug  2 17:59 helloworld.js
```

Reply:
Actually this case I can remove this file since I own the containing directory, it just warns me:

```
$ rm helloworld.js 
$ rm: remove write-protected regular file 'helloworld.js'? 
```

But in the general case if you run `make` or `ninja` in the container without `-u` it creates directories and contents that cannot be removed without sudo.

Reply:
In any case, the `-u` is present in all the other documentation so I think it would be weird not to also include it here.   If we decide its not needed or we can find an alternative workaround we can update this everywhere, but I'm just copying and pasting from our existing documentation here.

Reply:
Ok, I don't have strict opinion, was just trying to keep examples simple, but your arguments make sense :)